rsts/concepts/admin.rst

```diff
@@ -59,7 +59,7 @@ The managers utilize additional components to process requests. These additional
 Repository
 ----------
 Serialized entities (tasks, workflows, launch plans) and executions (workflow-, node- and task-) are stored as protos defined
-`here <https://github.com/flyteorg/flyteidl/tree/master/protos/flyteidl/admin>`__.
+`here <admin>`__.
```

Comment:
Which link is this pointing to?

Reply:
oh its not pointing correctly now. I remember seeing flyteidl having a reference in intersphinx mapping page. Fixing it now.